modules/png-extraction/ImageExtractor.py

```diff
@@ -128,7 +142,8 @@ def extract_headers(f_list_elem):
     # dicom images should not have more than 300
     if len(kv)>500:
         logging.debug(str(len(kv)) + " dicoms produced by " + ff)
-    kv.append(('file',chunk[nn])) #adds my custom field with the original filepath
+    # kv.append(('file',chunk[nn])) #adds my custom field with the original filepath
```

Comment:
What is this change?

Reply:
The chunk variable was getting initialized to zero if I used global keyword
Therefore I checked what it was receiving and passed the necessary variable from execute_png_extractor funcion.

Reply:
ok, then you can remove the line: 
# kv.append(('file',chunk[nn])) #adds my custom field with the original filepath
since it is just a code comment.

Reply:
ok, will do that